recipe/bld.bat

```diff
@@ -15,6 +15,8 @@ cmake .. ^
       -DCMAKE_CXX_STANDARD=17 ^
       -DCMAKE_PREFIX_PATH=%CONDA_PREFIX% ^
       -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Comment:
Seems like this may need to be fixed upstream (presumably dllimport/dllexport annotations are missing)?

Reply:
Yes, but those are really invasive and that's why many projects don't have them 🤷

Reply:
more practically, google never fixed the protobuf compiler to be able to insert them properly so I guess it's not possible to fix anyways. (at least the last time I took a swing at that.)